lib/editor/util/timetable.js

```diff
@@ -40,12 +40,27 @@ export function getHeaderColumns (
   return columns.filter(c => c.type !== 'DEPARTURE_TIME')
 }
 
-export function parseTime (timeString: string) {
-  const date = moment().startOf('day').format('YYYY-MM-DD')
-  return moment(date + 'T' + timeString, TIMETABLE_FORMATS).diff(
-    date,
-    'seconds'
-  )
+/**
+* Handles pasted data from clipboard (e.g. from CSV file)
+* If departure/arrival time cell, pastes in time format, otherwise returns string as is
+ */
+let alertShown = false
```

Comment:
This is functioning as a constant outside of the method. Should this be inside `parseCellValue` on the first line?

Reply:
although whenever i define my variable inside the function it creates an infinite loop where alerts keep showing ... i might be missing something